### ISSUE TITLE
VA-alert - Icons need alt text

### ISSUE THREAD

**Issue by aagosto90** [department-of-veterans-affairs/vets-design-system-documentation#979]
Check the [design system backlog](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues?q=is%3Aopen+is%3Aissue+label%3Apattern-new%2Ccomponent-new%2Cpattern-update%2Ccomponent-update) and existing [components](https://design.va.gov/components/) and [patterns](https://design.va.gov/patterns/) to see if your proposal has already been suggested. 

## What
> For the va-alert components, there should be alt text for the preceding icons for screen readers. Currently the icons are using aria-hidden so they don't show up for the screen readers.

![Screen Shot 2022-06-23 at 12 05 44 PM](https://user-images.githubusercontent.com/97965610/175344585-0b78825e-be7e-48c7-93f4-6e59b04bdf69.png)

## Why
> If the icons are necessary and/or helpful for sighted users, then the alt text would be helpful to screen reader users as well. If this is left up to individual teams, they would have to use a span class with sr-only class inside the VA Alert and come up with their own alt text, which will result in discrepancies across the site.

## Next steps
You may present your work to the Design System Council at an upcoming meeting. If you do not or cannot attend the Design Council Meeting, you can opt to get an asynchronous approval. 

Submit requests to join an upcoming Design System Council meeting in #platform-design-system.  

During the meeting, the Design System Council Working Group will evaluate the request and make a decision. 

If your request is approved, you can [add your component or pattern to the system](https://design.va.gov/about/contributing-to-the-design-system#4-add-your-component-or-pattern-to-the-system). If you have any questions on how to add your component or pattern to the system, please reach out to the Design System Team at #platform-design-system.

**Comment by caw310** [comment]
@SarahKay8 , would you please test to confirm this is an issue?

**Comment by davidakennedy** [comment]
Thanks for bringing this up!

Broadly speaking, from the [example alerts](https://design.va.gov/components/alert) we have in the design system, I would say that the icons in use are decorative. If you were to take the icons away, no context would be lost. That's dependent on the text in the alert, especially the heading. They do provide visual interest, but to screen reader users with some vision or no vision, it adds noise to the experience.

So the current way we handle the SVG icons, hiding them from screen readers, makes sense. I wouldn't recommend this change unless I'm missing context for how alerts are getting used in the wild. Much of this, as I said, depends on the text of the alert. In general, the use cases where icons need to have descriptions would be stand-alone icons that are interactive. For example, an x to close a dialog or a social media icon all by itself.

All that said, if we have use cases in the wild where the icons could stand alone without text, or we have feedback coming from Veterans that they're not getting the gravity/purpose of the alerts, perhaps changing this would be a good thing to explore.
